integrations/hermes/marmot/adapter.py

```
@@ -2250,57 +2261,199 @@ async def _send_media_upload(
+                            exc,
+                        )
+                        await asyncio.sleep(SEND_MEDIA_RETRY_BACKOFF_S[attempt])
+                        continue
+                    logger.debug("Marmot send_media failed: %s", exc)
+                    return SendResult(success=False, error=str(exc), retryable=is_retryable(exc))
+            return SendResult(
+                success=False,
+                error=str(last_exc) if last_exc else "Marmot media send failed",
+                retryable=is_retryable(last_exc) if last_exc else False,
+            )
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)